pom.xml

```diff
@@ -551,15 +551,15 @@
       org.apache.commons.vfs2.*;resolution:=optional,
       org.springframework.*;resolution:=optional,
       com.fasterxml.jackson.*;resolution:=optional,
-      org.yaml.snakeyaml.*;resolution:=optional,
+      org.snakeyaml.engine.*;resolution:=optional,
       *
     </commons.osgi.import>
     <log4j.version>2.19.0</log4j.version>
     <slf4j.version>2.0.5</slf4j.version>
     <!-- Spring 6 requires Java 17 -->
     <spring.version>5.3.24</spring.version>
 
-    <japicmp.skip>false</japicmp.skip>
+    <japicmp.skip>true</japicmp.skip>

```

Comment:
-1 obviously, you can't turn off this check for binary compatibility.

Reply:
it is binary incompatible

Reply:
Yes: Don't change that setting. You cannot break binary compatibility (BC) outside of a major release upgrade. If PR breaks BC, then it's -1.

Gary

Reply:
@garydgregory how can I indicate that it is a backwards incompatible change ? To be included in the next major release. I think the master branch is exactly for it

Reply:
You don't indicate anything, and you don't turn off JApiCmd as JApiCmd will fail the build if the PR breaks BC.

Reply:
@garydgregory I did not quite catch you. Do you mean that I need to provide a PR which breaks the build ?